crates/matrix-sdk-indexeddb/src/lib.rs

```diff
@@ -12,3 +15,56 @@ mod cryptostore;
 pub use cryptostore::IndexeddbStore as CryptoStore;
 #[cfg(target_arch = "wasm32")]
 pub use state_store::IndexeddbStore as StateStore;
+
+#[cfg(target_arch = "wasm32")]
+#[cfg(feature = "encryption")]
+/// Create a [`StateStore`] and a [`CryptoStore`] that use the same name and
+/// passphrase.
+pub async fn open_stores_with_name(
```

Comment:
is there any use case where someone would need to access this function instead of the other? I'd rather make this private.

```suggestion
/// Create a [`StateStore`] and a [`CryptoStore`] that use the same name and
/// passphrase.
async fn open_stores_with_name(
```